src/common/TrackedOp.cc

```diff
@@ -88,8 +88,10 @@ void OpHistory::_insert_delayed(const utime_t& now, TrackedOpRef op)
   double opduration = op->get_duration();
   duration.insert(make_pair(opduration, op));
   arrived.insert(make_pair(op->get_initiated(), op));
-  if (opduration >= history_slow_op_threshold.load())
+  if (opduration >= history_slow_op_threshold.load()) {
     slow_op.insert(make_pair(op->get_initiated(), op));
+    slow_ops_count++;
```

Comment:
We can use `slow_op.size()` instead. That way `slow_ops` count after removals will also be accurate.

Reply:
I want to count the number of slow op since OSD started running. If we use slow_op.size(), the most number of ops in slow_ops is 20.